CHANGELOG.rst

```diff
@@ -6,7 +6,10 @@ Version 3.0.2
 - Fix 'raw' mode in ``neurom stats``.
 - Add example astrocyte analysis notebook.
 - Fix readthedocs documentation build.
-- Delete all requirements txt files and update documentation accordingly
+- Delete all requirements txt files and update documentation accordingly.
+- Add ``neurom.features.morphology.trunk_angles_inter_types`` and
```

Comment:
Considering that the new API is added, I recommend rename `Version 3.0.2` to `Version 3.1.0` in Changelog.

Reply:
Good point, I changed